<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QDialog" name="CreditView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>750</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="wholeAmount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая сумма кредита:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="creditSum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="durationText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок:</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>12.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="typeOfPays">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип ежемесячного платежа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="monPay">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Ежемесячный платеж:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="monthlyPay">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="overpaymentText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Переплата по кредиту:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="overpayment">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="fullSum">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="yearDur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>В годах</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">durationType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="monthDur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>В месяцах</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">durationType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="ann">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">payType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="diff">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">payType</string>
   </attribute>
  </widget>
  <widget class="QSpinBox" name="duration">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>21</width>
     <height>471</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>18</red>
         <green>69</green>
         <blue>89</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="216">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Ayuthaya</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">Line {
            &quot;color: #AAAAAA;
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>491</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="payType"/>
  <buttongroup name="durationType"/>
 </buttongroups>
</ui>
